manifests/init.pp

```diff
@@ -217,9 +217,6 @@
   validate_string($fastcgi_cache_keys_zone)
   validate_string($fastcgi_cache_max_size)
   validate_string($fastcgi_cache_inactive)
-  if ($fastcgi_cache_key != false) {
```

Comment:
@bastelfreak You won't forget to add the validation back in in another PR?  Or maybe you should just put the new validation in using a data type in this PR too.  Users don't really care how the validation is performed.